RestaurantReservation.Db/Interfaces/IEmployeeRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Interfaces
+{
+    public interface IEmployeeRepository
+    {
+        Task CreateEmployeeAsync(Employee employee);
+        Task<Employee> GetEmployeeAsync(int employeeId);
+        Task<IEnumerable<Employee>> GetAllEmployeesAsync();
+        Task UpdateEmployeeAsync(Employee employee);
+        Task DeleteEmployeeAsync(int employeeId);
+        Task<IEnumerable<Employee>> ListManagersAsync();
```

Comment:
Can We name it as GetManagersAsync()

Reply:
Sure! I've just committed to the requirement that it written `ListManagers()`